src/back/glsl/mod.rs

```diff
@@ -1551,7 +1551,10 @@ impl<'a, W: Write> Writer<'a, W> {
                     None
                 };
 
-                write!(self.out, "{}(", function.unwrap_or(""))?;
```

Comment:
what happens if we have `(4+5) * 6` today? would this now become `4 + 5 * 6`?

Reply:
Nice catch! Thank you!

Maybe I should find another way how we can improve readability :smile: